containers/build.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python
+
+from argparse import ArgumentParser
+import hashlib
+import logging
+import os
+from pathlib import Path
+import shutil
+import subprocess
+
+import requests
+import yaml
+
+# Logging configuration
+root = logging.getLogger()
+if not root.handlers:
+    ch = logging.StreamHandler()
+    formatter = logging.Formatter(
+        "%(asctime)s - %(name)s - %(levelname)s - %(message)s"
+    )
+    ch.setFormatter(formatter)
+    root.addHandler(ch)
+
+logger = logging.getLogger("build")
+logger.setLevel(logging.INFO)
+
+# Build parameters
+CONTAINER_NAMES = ["mlflow"]
```

Comment:
This is excellent because as we develop we develop this project, we'll start working under a regime called GitOps. In theory, a git repo should perfectly describe a infrastructure (code as infrastructure). 

Counterintuitively, the more config that is baked into the code the better. I like how you did this and not made it part of the argument parser for this reason.